src/main/scala/io/iohk/ethereum/db/storage/TransactionMappingStorage.scala

```diff
@@ -0,0 +1,29 @@
+package io.iohk.ethereum.db.storage
+
+import java.nio.ByteBuffer
+
+import akka.util.ByteString
+import io.iohk.ethereum.db.dataSource.DataSource
+import io.iohk.ethereum.db.storage.TransactionMappingStorage.{TransactionLocation, TxHash}
+import boopickle.Default._
+
+class TransactionMappingStorage(val dataSource: DataSource) extends KeyValueStorage[TxHash, TransactionLocation, TransactionMappingStorage] {
+
+  val namespace: IndexedSeq[Byte] = Namespaces.TransactionMappingNamespace
+  def keySerializer: TxHash => IndexedSeq[Byte] = identity
+  def valueSerializer: TransactionLocation => IndexedSeq[Byte] = Pickle.intoBytes(_).array()
```

Comment:
That was my first idea but as there is a task for removing the use of it for storages (https://iohk.myjetbrains.com/youtrack/issue/EC-119), `boopickle` serializing is also used for `FastSyncStateStorage` and it required the same effort decided to use the `boopickle` serializing.